docs/schema.md

```diff
@@ -0,0 +1,186 @@
+
+```java
+final Properties props = new Properties();
+props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, KafkaAvroSerializer.class);
+props.put(KafkaAvroSerializerConfig.SCHEMA_REGISTRY_URL_CONFIG, "http://localhost:8001");
+
+final String topic = "my-avro-kafka-topic";
+final KafkaProducer<String, User> producer = new KafkaProducer<>(props);
+RecordMetadata metadata = producer.send(new ProducerRecord<>(topic, new User("alice", 10))).get();
+System.out.println("Sent to " + metadata);
+producer.close();
+```
+
+You will see the following output:
+
+```
+Sent to my-avro-kafka-topic-0@0
+```
+
+Run the following consumer application:
+
+```java
+final Properties props = new Properties();
+props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, KafkaAvroDeserializer.class);
+props.put(ConsumerConfig.GROUP_ID_CONFIG, "my-group");
+props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+props.put(KafkaAvroSerializerConfig.SCHEMA_REGISTRY_URL_CONFIG, "http://localhost:8001");
+
+final KafkaConsumer<String, User> consumer = new KafkaConsumer<>(props);
+consumer.subscribe(Collections.singleton("my-avro-kafka-topic"));
+while (true) {
+    ConsumerRecords<String, User> records = consumer.poll(Duration.ofSeconds(1));
+    records.forEach(record -> System.out.println("Received " + record.value() + " from "
+            + record.topic() + "-" + record.partition() + "@" + record.offset()));
+    if (!records.isEmpty()) break;
+}
+consumer.close();
+```
+
+You will see the following output:
+
+```
+Received {"name": "alice", "age": 10} from my-avro-kafka-topic-0@0
+```
+
+### Query the created schema
+
+Query the schema whose id is 1 (i.e. the 1st schema):
+
+```bash
+$ curl -L http://localhost:8001/schemas/ids/1; echo
+{
+  "schema" : "{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"example.avro\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"age\",\"type\":\"int\"}]}"
+}
+```
+
+Query the subjects:
+
+```bash
+$ curl -L http://localhost:8001/subjects; echo
+[ "my-avro-kafka-topic-value" ]
+$ curl -L http://localhost:8001/subjects/my-avro-kafka-topic-value/versions; echo
+[ 1 ]
+$ curl -L http://localhost:8001/subjects/my-avro-kafka-topic-value/versions/1; echo
+{
+  "id" : 1,
+  "schema" : "{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"example.avro\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"age\",\"type\":\"int\"}]}",
+  "subject" : "my-avro-kafka-topic-value",
+  "version" : 1,
+  "type" : "AVRO"
+}
+```
+
+See [here](https://docs.confluent.io/platform/current/schema-registry/schema_registry_tutorial.html#terminology-review) for a quick start of Confluent's Schema concepts.
```

Comment:
```suggestion
For a quick start of Confluent's Schema concepts, See [Confluent documentation](https://docs.confluent.io/platform/current/schema-registry/schema_registry_tutorial.html#terminology-review) .
```
If this sentence belongs to the query created schema section, it is OK to put the sentence here. If it is just a reference that you want users to get more info, it would be great to use a format, such as:
## References
For a quick start of Confluent's Schema concepts, See [Confluent documentation](https://docs.confluent.io/platform/current/schema-registry/schema_registry_tutorial.html#terminology-review) .